### PR TITLE
Tweak server configurations.

### PR DESCRIPTION
**Pull request by rabbah**
Enable auto-timeouts. Remove unnecessary server configuration in invoker.

### CODE REVIEW

File: core/controller/src/main/resources/application.conf [hunk 1]
```diff
@@ -2,10 +2,38 @@
 include "logging"
 
 # see http://spray.io/documentation/1.2.2/spray-can/configuration/
```
Comment (markusthoemmes) [inline core/controller/src/main/resources/application.conf]: Please update the dead link to http://spray.io/documentation/spray-can/configuration/

**Comment by rabbah** [comment]
PG2/986 approved.

**Comment by rabbah** [comment]
@markusthoemmes you'll want to review this again - I made some changes to limit the entity names.

**Comment by rabbah** [comment]
@markusthoemmes you'll want to review the last two commit (limiting entity name and test for max url path).

PG3/230 for the latest commits.

**Comment by rabbah** [comment]
pg3/234 for the latest.

File: ansible/roles/nginx/templates/nginx.conf.j2 [hunk 2]
```diff
@@ -6,7 +6,7 @@ events {
 }
 
 http {
-{# temporary hack to allow large uploads, need to thread proper limit into here #}
+{# allow large uploads, need to thread proper limit into here #}
```
Comment (rabbah) [inline ansible/roles/nginx/templates/nginx.conf.j2]: we've had it for a while - still think it's temporary?

File: core/controller/src/main/resources/application.conf [hunk 3]
```diff
@@ -1,11 +1,40 @@
...
+  # Set to `infinite` to completely disable idle connection timeouts.
+  #
+  # Explaining the set value:
+  # This must be greater than the request timeout.
+  idle-timeout = 120s
+
   parsing {
+    # This indirectly puts a bound on the name of entities
+    # 8k matches nginx default
+    max-uri-length = 8k
```
Comment (rabbah) [inline core/controller/src/main/resources/application.conf]: this will match edge/router.

**Pull request merged**